internal/coremsgs/en_struct_descriptions.go

```diff
@@ -161,6 +161,7 @@ var (
 	TransactionType          = ffm("Transaction.type", "The type of the FireFly transaction")
 	TransactionNamespace     = ffm("Transaction.namespace", "The namespace of the FireFly transaction")
 	TransactionCreated       = ffm("Transaction.created", "The time the transaction was created on this node. Note the transaction is individually created with the same UUID on each participant in the FireFly transaction")
+	TransactionBlockchainID  = ffm("Transaction.blockchainId", "The blockchain transaction ID, in the format specific to the blockchain involved in the transaction. Not all FireFly transactions include a blockchain")
```

Comment:
It looks like this message is a copy of the one below it. Was it intentional to not include the last half of the string, and not end the sentence with a period?

Reply:
Yes. Seems like every message in this file omits the final period (not sure if there's a reason, but that's how it looks). I omitted the last bit since it seemed to apply specifically to the plural field (ie listing multiple blockchain IDs).